identifier/identifier.go

```diff
@@ -48,6 +75,16 @@ func NewDNS(domain string) ACMEIdentifier {
 	}
 }
 
+// SliceNewDNS is a convenience function for creating a slice of ACMEIdentifiers
+// with Type "dns" for a given slice of domain names.
+func SliceNewDNS(domains []string) []ACMEIdentifier {
```

Comment:
If you make this
```suggestion
func SliceNewDNS(domains ...string) []ACMEIdentifier {
```
then everywhere you call it, you can just pass it a bunch of individual strings like `identifier.SliceNewDNS("hello", "world")` instead of having to package them into a slice like `identifier.SliceNewDNS([]string{"hello", "world"})`.

Reply:
From our convo: actually I think this can be almost entirely replaced by the combination of
- SliceFromProto(idents []ACMEIdentifier, dnsNames []string), for when extracting from the two fields of a proto message;
- SliceFromCert(x509.Certificate); and
- csr.IdentsFromCSR(x509.CertificateRequest), upgraded from csr.NamesFromCSR.